src/pymor/reductors/neural_network.py

```diff
@@ -423,13 +495,19 @@ class NeuralNetworkStatefreeOutputReductor(NeuralNetworkReductor):
         The full-order |Model| to reduce. If `None`, the `training_set` has
         to consist of pairs of |parameter values| and corresponding outputs.
     training_set
-        Set of |parameter values| to use for POD and training of the
-        neural network. If `fom` is `None`, the `training_set` has
-        to consist of pairs of |parameter values| and corresponding outputs.
+        List of |Parameter values| to use for POD and training of the
```

Comment:
```suggestion
        List of |Parameter values| to use for training of the
```
That was also wrong before...